packages/core/src/common/abstractPureComponent.ts

```diff
@@ -30,14 +30,14 @@ export abstract class AbstractPureComponent<P, S = {}> extends React.PureCompone
 
     constructor(props?: P, context?: any) {
         super(props, context);
-        if (!isNodeEnv("production")) {
+        if (!isNodeEnv('production')) {
             this.validateProps(this.props);
         }
     }
 
-    public componentWillReceiveProps(nextProps: P & { children?: React.ReactNode }) {
```

Comment:
same here about keeping this deprecated method around for backwards compatibility